layers/+lang/typescript/README.org

```diff
@@ -45,17 +48,32 @@ file.
 (setq-default dotspacemacs-configuration-layers '(typescript))
 #+END_SRC
 
-or if you don't need linting.
+If you don't need linting.
```

Comment:
Use a `:` at the end of the line. The phrasing can also be a little bit improved, like «If you don't need linting, you can use instead:».